pkg/apis/crds/karpenter.sh_nodepools.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.16.2
```

Comment:
I think these may be updated now so you may just need to rebase this PR based off of the current main